testing/python_mldb_interface_test.py

```diff
@@ -0,0 +1,92 @@
+#
+# python_mldb_interface_test.py
+# Mich, 2016-01-25
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+import unittest
+
+if False:
+    mldb_wrapper = None
+
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+
+class PythonMldbInterfaceTest(unittest.TestCase):
+
+    def test_log(self):
+        mldb.log("Testing log")
+
+    def test_get(self):
+        res = mldb.get('/ping')
+        mldb.log(res)
+        mldb.log(res.text)
+        mldb.log(type(res))
+        mldb.log(dir(res))
+
+        with self.assertRaises(mldb_wrapper.ResponseException):
+            mldb.get("/unexisting")
```

Comment:
can we get a small test case that shows the contents of the `ResponseException` and how to access them?